pages/index.jsx

```diff
@@ -8,10 +8,14 @@ import Pontoon from '../components/pontoon.jsx';
 var Index = React.createClass({
   render: function() {
     var metaData = this.props.metaData;
+    var robots = 'index, follow';
     var googleFonts = "https://fonts.googleapis.com/css?family=Open+Sans:600,400,300,300italic";
     if (this.props.localeInfo === "cs") {
       googleFonts += "&subset=latin-ext";
     }
+    if (metaData.current_url.indexOf('thank-you') !== -1) {
```

Comment:
Can we make this check `>= 1` or `> 0`? The `!== -1`? I think it might be easier to read.